ROSCO_toolbox/controller.py

```diff
@@ -186,6 +186,8 @@ def tune_controller(self, turbine):
         Ng = turbine.Ng                         # Gearbox ratio (-)
         rated_rotor_speed = turbine.rated_rotor_speed               # Rated rotor speed (rad/s)
 
+        # ------------- Saturation Limits --------------- #
+        turbine.max_torque = turbine.rated_torque * self.controller_params['max_torque_factor']
```

Comment:
Should this be made `controller.max_torque`? This technically might be more of a controller parameter than turbine parameter, though maybe this is just semantics.

Reply:
It was originally `turbine.max_torque`, so I kept it and moved it to `controller.py` where it had access to `controller_params` rather than changing the name throughout.